docs/components/receive.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Receiver
+type: docs
+menu: components
+---
+
+# Receiver
+
+The `thanos receive` command implements the [Prometheus Remote Write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write). It builds on top of existing Prometheus servers and retains their usefulness while extending their functionality with long-term-storage, horizontal scalability, and downsampling. The [Thanos Sidecar](./sidecar.md) is not sufficient for this, as the system would always lag the block length behind (typically 2 hours), which would prevent the most common query pattern of Prometheus: real-time queries of very recent data.
```

Comment:
I don't think the last sentence describes this particularly good.
I'm not even fully grasping what you're trying to say.

Reply:
I've cheated and copied that over proposal document 😅 Happy to remove or paraphrase it.

Reply:
Hmm for the sentence starting with `It builds on top of existing Prometheus servers...` I think is a little bit fuzzy for people new to Thanos and who aren't familiar with everything. Also if Thanos builds on top of Prometheus. Maybe we should collaborate on a Google doc so we can iterate faster. Wdyt?

Reply:
@squat Yes, please. Greate idea. https://docs.google.com/document/d/1H4psm3ancNRyUfg4UilroAEKb96zm_dldH6UnVoLB6A/edit#